_sc2data/Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml

```diff
@@ -2086,4 +2085,55 @@
         <PlanetPanelCamera value="Star2CameraMid04"/>
         <ShadowRadius value="0.541500"/>
     </CModel>
+    <CModel id="AP_AberrationPortrait" parent="Portrait" Race="Zerg">
+        <Model value="Assets\Portraits\Zerg\AberrationPortrait\AberrationPortrait.m3"/>
+        <FacialController value="LocalizedData\Facial\AberrationPortrait.fxa"/>
+        <Flags index="Facial" value="1"/>
+        <Image value="Assets\Textures\AberrationPortrait_Static.dds"/>
+        <Lighting value="AberrationPortrait"/>

```

Comment:
I think you need to copy the light as `AP_AberrationPortrait`. This can be easily missed

Reply:
Solved in [5fae243](https://github.com/Ziktofel/Archipelago/pull/22/commits/5fae24302ddaf6eb0bc5d77905411b6c546e0a63)